core/src/test/java/tech/pegasys/eth2signer/core/multikey/DirectoryBackedArtifactSignerProviderTest.java

```diff
@@ -83,6 +81,7 @@ void signerReturnedForValidMetadataFile() throws IOException {
     final String filename = PUBLIC_KEY1;
     createFileInConfigsDirectory(filename);
     when(signerParser.parse(any())).thenReturn(artifactSigner);
+    signerProvider.loadSigners();
```

Comment:
can this be moved into the setup?

Reply:
I _THINK_ no, as the signerParser.parse() needs to be configured prior to loading.